workspaces/util/src/__tests__/timestampToYearMonthDay-test.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { timestampToYearMonthDay } from "../timestampToYearMonthDay";
+
+describe("timestampToYearMonthDay", () => {
+  it.each([
+    [1721211, "1970-01-20"],
```

Comment:
A few more things to include:

* 0 -- I'd like to see that it's 1970-01-01
* a negative number -- it doesn't happen that often, but we may want a date before 1970 at some point, it could be nice to have it documented that this works
* a large enough number that the year is 3000, or more than 4 digits
* a non-integer -- I assume this will be rounded or truncated, but could be good to have tried